### PR TITLE
Commented out rule in backend.xml that caused the scripts being pulle…

### PR DESCRIPTION
**Pull request by sneridagh**
…d twice in case that you are using it to style backend views.

### CODE REVIEW

**Comment by sneridagh** [comment]
This fixes https://github.com/plone/plonetheme.barceloneta/issues/158

**Comment by sneridagh** [comment]
@mauritsvanrees backend.xml should be also used (which doesn't happen in p.barceloneta, because backend is already themed)... In fact in barceloneta the fix is more a cosmetic one than really used. So if someone comes to barceloneta and wants to looks how things are done, it doesn't get tricked.

The one in bobtemplates.plone is the important one, because it's what people will use to make their themes => https://github.com/plone/bobtemplates.plone/pull/321

File: CHANGES.rst [hunk 1]
```diff
@@ -15,7 +15,12 @@ New features:
 
 Bug fixes:
 
-- *add item here*
+- Commented out rule in backend.xml that caused the scripts being pulled twice,
```
[the remark this answers is not in the document]
Comment (jensens) [reply CHANGES.rst]:
```suggestion
- Commented out ``<after css:theme-children="head" css:content="head script" />`` in backend.xml that caused the scripts being pulled twice,
```

[the remark this answers is not in the document]
Comment (sneridagh) [reply CHANGES.rst]: 👍

**Comment by jensens** [comment]
@jenkins-plone-org please run jobs

**Comment by mauritsvanrees** [comment]
This got merged. Or rather, it looks like @agitator took the last commit from this branch and created the 2.x branch from it today.
Should be okay, the diff compared with the last release is only the change above. Should Jenkins start failing with this, we can revert.
The news snippet is missing, but I will copy it.

**Pull request closed without merging**